### ISSUE TITLE
Update the demo

### ISSUE THREAD

**Issue by cristan** [smaldini/grails-events-push#54]
The reason why I'm going to try another plugin now is because I can't get the demo to work. First of all, it is annoying that it targets grails 3.0.0-snapshot. Also, the version of grails-events-push is events-push:1.0.0.BUILD-SNAPSHOT in stead of a proper version like events-push:1.0.M7. I now get "Cannot find the class file for org.hamcrest.Matcher. Fix the build path then try building this project" and I'm giving up now.

**Comment by longwa** [comment]
Yeah, it seems like this plugin is in a major transitional period right now. The 1.0.0 snapshot is using Reactor, Platform Core doesn't seem to work with Grails 2.3.x, and the documentation is all for the prior version. The only way to find examples is to look at the sample *Events.groovy files in the source.

We had a working version of events-push in Grails 2.2.4 but I've been unable to get the push stuff working again in 2.3.5 using the new 1.0.0-Snapshot. It looks like it should be working, but the events never make it to the browser.

**Comment by cristan** [comment]
Ok, well it's nice to know you're working on it. To be fair, I didn't get any other of the push library's to work as well. Thanks for the reply :)

**Comment by longwa** [comment]
We have events-push working in 2.2.4 using the platform-core stuff. I've not been able to get it working using the new grails-events stuff in the latest version.

It would awesome if @smaldini would update the grailsTodo app to be a working version with the latest events-push snapshot, then at least I could base my changes off it it.

**Comment by brewathome** [comment]
+1. The current version of grailsTodo does not work for me. Do you know if anyone is working on this? I am using 2.3.8, BTW.

**Comment by jmhobrien** [comment]
I just tried with grails 2.3.2 with no luck. Please fix.

**Comment by ghost** [comment]
hiii 
does anyOne tell me the step by step with their explanation how to use the event push plugin with assets-pipeline in grails 2.4.3.
Actually i am new to grails but it find quiet interesting to implement this plug-in ..
i read the documentation including some surffing stuff but doesn't get to much..
wat is the saveto do things .. it seems like it a event
i tried to implement it but the thing event is not reconise in the controller or getting error
 the step by step how to use pu
Actually wat i want is to totally understand the mechnism how things going on .so that i can applied it too my project

any help ,will b appriciated
thanks in advance
